pkg/nonk8sapi/policies/list_status_test.go

```diff
@@ -0,0 +1,52 @@
+package policies
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/gin-gonic/gin"
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPolicyStatusList(t *testing.T) {
+	// use the js encodeURIComponent() to encode password
+	postgreUrl := "postgresql://hoh-process-user:password@host:5432/hoh"
```

Comment:
Is this a fake DB connection? or does the UT need to run against real DB?

Reply:
Yeah. It's a fake DB URL.